test/distributed/_tensor/test_attention.py

```diff
@@ -289,6 +294,7 @@ def test_ring_attention_native_transformer(self, is_causal: bool) -> None:
     @with_comms
     @sdpa_kernel(backends=[SDPBackend.FLASH_ATTENTION])
     def test_ring_attention_custom_transformer(self) -> None:
+        _cp_options.rotate_method = _RotateMethod.ALL_TO_ALL
```

Comment:
should all the CP tests be parametrized and test both rotate methods? or what reason do we prefer to only test A2A in some cases